toolchain/check/facet_type.h

```diff
@@ -39,10 +39,10 @@ auto GetImplWitnessAccessWithoutSubstitution(Context& context,
                                              SemIR::InstId inst_id)
     -> SemIR::InstId;
 
-// Creates a impl witness instruction for a facet type. The facet type is
-// required to be complete if `is_definition` is true or the facet type has
-// rewrites. Otherwise a placeholder witness is created, and
-// `AllocateFacetTypeImplWitness` can be used at the `impl` definition.
+// Creates a impl witness instruction for a facet type. A placeholder witness
+// table is created if there are no rewrites in the declaration, and
+// `AllocateFacetTypeImplWitness` will construct the table of the appropriate
```

Comment:
`AllocateFacetTypeImplWitness` is removed in this PR.

Reply:
This comment is moved and fixed in https://github.com/carbon-language/carbon-lang/pull/6484. Copying over some of those fixes here so the comment isn't left incorrect in between.